tests/baselines/reference/abstractPropertyNegative.errors.txt

```diff
@@ -38,13 +38,10 @@ tests/cases/compiler/abstractPropertyNegative.ts(41,18): error TS2676: Accessors
     }
     class C extends B {
           ~
-!!! error TS2515: Non-abstract class 'C' does not implement inherited abstract member 'm' from class 'B'.
-          ~
 !!! error TS2515: Non-abstract class 'C' does not implement inherited abstract member 'mismatch' from class 'B'.
-          ~
-!!! error TS2515: Non-abstract class 'C' does not implement inherited abstract member 'prop' from class 'B'.
-          ~
-!!! error TS2515: Non-abstract class 'C' does not implement inherited abstract member 'readonlyProp' from class 'B'.
+!!! error TS2515:   Non-abstract class 'C' does not implement inherited abstract member 'm' from class 'B'.

```

Comment:
this is what I was implementing towards
```
!!! error TS<NEWCODE>: Non-abstract class 'C' does not implement every abstract members of class 'B'
!!! error TS<NEWCODE>:   Class 'C' is missing the following member implementations from class 'B': prop, readonlyprop, m
```
- new generic error with details in span

Reply:
But if this implementation works for us, I'll stop what I was doing; else I should be able to finish in a day or two

Reply:
I think your message is more suitable for the proposal.
> Proposal
> Instead of generating two errors in this case, generate a single, generic diagnostic that states > something like: B does not implement all abstract members of A and on that diagnostic return > related spans that state specifically which members are not implemented (b and c).

Please notice us your worked branch and let's ask to @DanielRosenwasser